docs/setup-steps-1.4.0.md

```diff
@@ -0,0 +1,265 @@
+# 1.4.0
+
+## Pre-requisites
+
+CTK should be deployed with the required dockers.
+
+* compliance-toolkit-service: 1.4.0
+* compliance-toolkit-ui: 1.4.0
+* compliance-toolkit-batch-job: 1.4.0
+
+To deploy Compliance Toolkit, we require the below mandatory services:
+
+## Dependent Service (dockers)
+
+* Artifactory: mosipid/artifactory-ref-impl: 1.2.0.1-B2
+* Audit manager: mosipid/kernel-auditmanager-service: 1.2.0.1-B1
+* Auth Manager: mosipid/kernel-authmanager: 1.2.0.1-B1
+* Key Manager: modipid/kernel-keymanager-service: 1.2.0.1-B1
+* Partner Management: mosipid/partner-management-service: 1.2.0.1-B1
+* KeyCloak: mosipid/keycloak-init: 1.2.0.1-B1
+* Postgres: mosipid/postgres-init: 1.2.0.1-B1
+* Config Server: config-server: mosipid/config-server: 1.1.2
+* Notification Service: mosipid/kernel-notification-service: 1.2.0.1-B1
+* ClamAV: clamav/clamav: latest
+* MinIO
+
+**The Setup guide is a checklist for the three categories below:**
+
+1. Configuration checks
+2. Steps to load testdata, schemas, testcases and terms and conditions templates
+3. Steps to generate Android APK
+
+### Configuration checks
+
+1. Ensure that in the `kernel-default.properties`, the value of `mosip-toolkit-client` and `mosip-toolkit-android-client` is set in `auth.server.admin.allowed.audience`. If this was not set by default, then set it and restart `kernel-auth-service` and `compliance-toolkit-service`.
```

Comment:
Make sure that the kernel-default.properties file includes the mosip-toolkit-client and mosip-toolkit-android-client values in the auth.server.admin.allowed.audience setting. If these values are not set by default, configure them and then restart the kernel-auth-service and compliance-toolkit-service.